# Setpoint Mode Change Service

uint8 mode
---
bool success
uint8 result